projects/1.20/assets/dungeons-delight/dungeons_delight_gourmet_battles/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+{
+  "advancements.crispyandsweet.descr": "这些腿尝起来怎么样?",
+  "advancements.crispyandsweet.title": "香脆可口",
+  "advancements.dragon_delight.descr": "你烹饪了末地的守护者",
+  "advancements.dragon_delight.title": "龙之乐事",
+  "advancements.dungeon_apprentice.descr": "准备好开始地下城烹饪之旅了吗?",
+  "advancements.dungeon_apprentice.title": "地下城新手",
+  "advancements.flamingpudding.descr": "Sweet pain? hot sweet?甜蜜的痛苦？火热的甜蜜？",
+  "advancements.flamingpudding.title": "火焰布丁",
+  "advancements.ghostlyhanbuger.descr": "准备恶魂肉快餐！",
+  "advancements.ghostlyhanbuger.title": "恶魂汉堡",
+  "advancements.greenpizzeria.descr": "你做出了意大利奇迹!",
```

```suggestion
  "advancements.greenpizzeria.descr": "你做出了意大利奇迹！",
```

标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

感觉在neta文明6的奇观，不知道文明是怎么提示的